.circleci/config.yml

```diff
@@ -118,7 +106,7 @@ jobs:
             - run: yarn test
 
     test2.9:
-        docker: *docker-node-6
+        docker: *docker-node-
```

Comment:
missing `-8`, which might be tripping up circle in running the workflow

Reply:
Ha, thanks. I'm surprised Circle didn't give a better error indicator for the invalid reference...